util/topgen.py

```diff
@@ -30,7 +30,7 @@
 from topgen import get_hjsonobj_xbars
 from topgen import intermodule as im
 from topgen import lib as lib
-from topgen import merge_top, search_ips, strong_random, validate_top
+from topgen import secure_prng, merge_top, search_ips, strong_random, validate_top
```

Comment:
Do we still need `strong_random` here?

Reply:
This is still an open discussion point, but the probable answer is no.
i plan to do `strong_random` cleanup in a separate PR

Reply:
I've opened #19433 to track this.